dlp/snippets/inspect/inspect_test.go

```diff
@@ -276,3 +276,21 @@ func TestInspectBigquery(t *testing.T) {
 		})
 	}
 }
+
+func TestInspectWithCustomRegex(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	buf := new(bytes.Buffer)
```

Comment:
issue: don't need to use the `new` keyword here. Just use `var`.

Example:
```
var buf bytes.Buffer
```

Reply:
Nope, it doesn't require the `new` keyword. `var` is fine. Please fix.

Reply:
Yes, It doesn't require `new` keyword, So Fixed it Here -> https://github.com/GoogleCloudPlatform/golang-samples/blob/a2ad801c6209726d71b3d1fa670936e0fb7fdd30/dlp/snippets/inspect/inspect_test.go#L362